packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -102,25 +101,28 @@ export class DetailsHeaderBase extends React.Component<IDetailsHeaderBaseProps,
 
     this._events.on(selection, SELECTION_CHANGE, this._onSelectionChanged);
 
-    // We need to use native on this to prevent MarqueeSelection from handling the event before us.
-    this._events.on(this._rootElement!, 'mousedown', this._onRootMouseDown);
+    // this._rootElement.current will be null in tests using react-test-renderer
+    if (this._rootElement.current) {
+      // We need to use native on this to prevent MarqueeSelection from handling the event before us.
+      this._events.on(this._rootElement.current, 'mousedown', this._onRootMouseDown);
 
-    this._events.on(this._rootElement!, 'keydown', this._onRootKeyDown);
+      this._events.on(this._rootElement.current, 'keydown', this._onRootKeyDown);
 
-    if (this._getColumnReorderProps()) {
-      this._subscriptionObject = this._dragDropHelper.subscribe(
-        this._rootElement!,
-        this._events,
-        this._getHeaderDragDropOptions(),
-      );
+      if (this._getColumnReorderProps()) {
+        this._subscriptionObject = this._dragDropHelper.subscribe(
+          this._rootElement.current,
+          this._events,
+          this._getHeaderDragDropOptions(),
+        );
+      }
     }
   }
 
   public componentDidUpdate(prevProps: IDetailsHeaderBaseProps): void {
     if (this._getColumnReorderProps()) {
       if (!this._subscriptionObject) {
         this._subscriptionObject = this._dragDropHelper.subscribe(
-          this._rootElement!,
+          this._rootElement.current!,

```

Comment:
```suggestion
          this._rootElement.current,
```

Reply:
Unfortunately the ! is needed here since `current` isn't guaranteed by the typings to be defined.

Reply:
Though I suppose the proper way to fix that is wrapping in if()